### PR TITLE
Update Companion Screenshot

### PR DESCRIPTION
**Pull request by MeteorCoder**
### Description
This PR removes old Companion Screenshot referring to Makeroid. Adds a new Screenshot referring to now known as Kodular.

**Please make a new video on Google Play Store for the Kodular Companion and this screenshot can be added to Google Play Store to replace current ones referring to ex name.**

## Types of changes:
- [ ] Correct a spelling
- [x] Add new information
- [x] Change information

## Checklist:
- [x] I have proofread the changes. (Spellings, grammar, etc)
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs/live-development/index.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ Visit the below pages to learn how to connect the companion via different ways:
 
 **Step 2:** Open the app
 
-![](/assets/images/live-development/index-2.png)
+![](/assets/images/live-development/index-2.jpg)
```
Comment (barreeeiroo) [inline docs/live-development/index.md]: It should be .png
And crop the image to the borders of the mock phone

Comment (barreeeiroo) [inline docs/live-development/index.md]:
```suggestion
![](/assets/images/live-development/index-2.png)
```

Comment (MeteorCoder) [reply docs/live-development/index.md]: Fixed!

Comment (MeteorCoder) [reply docs/live-development/index.md]: Fixed before recognized.

**Pull request merged**